andela/AndelaEatsUI
# #162554964 Fix admin link on navbar

#### What does this PR do?

- Fix admin link on the navbar

#### Description of Task to be completed?

- create reducers, actions to get the user role from the endpoint
- implement functionality to ensure that only an admin can see the admin link in the navbar

#### How should this be manually tested?

- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `bg-admin-link-navbar-162554964`
- run these commands `npm run build` and `npm run dev` 
- Log in as a user and hover over the caret on the right of your profile picture. If you are an admin, you should see a link to the `Admin Dashboard`, if you are not, you won't see it.

#### What are the relevant pivotal tracker stories?
- [ #162554964](https://www.pivotaltracker.com/story/show/162554964)

#### Background Context

N/A

#### Screenshots (If Applicable)

N/A
